src/System.Runtime.Numerics/tests/BigInteger/modpow.cs

```diff
@@ -161,6 +161,23 @@ public static void ModPow2Large1SmallInt()
         }
 
         [Fact]
+        public static void ModPow3LargeInt()
+        {
+            byte[] tempByteArray1 = new byte[0];
+            byte[] tempByteArray2 = new byte[0];
+            byte[] tempByteArray3 = new byte[0];
```

Comment:
Why are these being initialized here?  Even in a test, seems like the allocations are unnecessary.